pom.xml

```diff
@@ -55,7 +55,7 @@
   <properties>
     <enforcer.skip>true</enforcer.skip> <!-- TODO numerous requireUpperBoundDeps, some probably indicative of real problems -->
     <jenkins.version>2.7</jenkins.version>
-    <findbugs-maven-plugin.version>3.0.2</findbugs-maven-plugin.version>
+    <findbugs-maven-plugin.version>3.0.5</findbugs-maven-plugin.version>
```

Comment:
This is to avoid
```
[ERROR] Failed to execute goal org.codehaus.mojo:findbugs-maven-plugin:3.0.2:findbugs (findbugs) on project nexus-jenkins-plugin: Unable to parse configuration of mojo org.codehaus.mojo:findbugs-maven-plugin:3.0.2:findbugs for parameter pluginArtifacts: Cannot assign configuration entry 'pluginArtifacts' with value '${plugin.artifacts}' of type java.util
.Collections.UnmodifiableRandomAccessList to property of type java.util.ArrayList
```
Fixed by https://github.com/gleclaire/findbugs-maven-plugin/pull/35 in findbugs-maven-plugin-3.0.4 (just upgraded to the latest version)